lib/smart_proxy_openscap/storage_fs.rb

```diff
@@ -23,7 +23,7 @@ def get_arf_html(digest)
       xml = get_arf_file(digest)[:xml]
       size = get_arf_file(digest)[:size]
       arf_object = OpenSCAP::DS::Arf.new(:content => xml, :path => 'arf.xml.bz2', :length => size)
-      arf_object.html.force_encoding('UTF-8')
+      arf_object.html
```

Comment:
Yeah, I now recall an issue with "ASCII-8BIT", which lead us to force_encoding.
I have monkey patched String for 1.8.